packages/astro/test/astro-styles-ssr.test.js

```diff
@@ -2,136 +2,141 @@ import { expect } from 'chai';
 import cheerio from 'cheerio';
 import { loadFixture } from './test-utils.js';
 
+let fixture;
+let index$;
+let bundledCSS;
+
+before(async () => {
+  fixture = await loadFixture({
+    projectRoot: './fixtures/astro-styles-ssr/',
+    renderers: ['@astrojs/renderer-react', '@astrojs/renderer-svelte', '@astrojs/renderer-vue'],
+  });
+  await fixture.build();
+
+  // get bundled CSS (will be hashed, hence DOM query)
+  const html = await fixture.readFile('/index.html');
+  index$ = cheerio.load(html);
+  const bundledCSSHREF = index$('link[rel=stylesheet][href^=assets/]').attr('href');
+  bundledCSS = await fixture.readFile(bundledCSSHREF.replace(/^\/?/, '/'));
+});
+
 describe('Styles SSR', () => {
-  let fixture;
+  describe('Astro styles', () => {
```

Comment:
Style SSR tests updated! Before we weren’t testing CSS contents, which were actually wrong on Windows. Now we are.

The confusing test that tested everything is now broken up by framework so failures are clearer.